From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jeremy Rose <jeremya@chromium.org>
Date: Mon, 3 May 2021 15:40:47 -0700
Subject: conditionally import strip_binary.gni in chromedriver tests

This import is conditional elsewhere (see e.g.
https://source.chromium.org/chromium/chromium/src/+/main:chrome/BUILD.gn;l=38;drc=dfe5d22212436b797a2f0dba23fe5152f4c5e871).
This removes a dependency on the //build/linux .gni files from the macos build.

Change-Id: Ia2f2c245f7182c9f48327962d9e8d37d0e9652b4

diff --git a/chrome/test/chromedriver/BUILD.gn b/chrome/test/chromedriver/BUILD.gn
index 31237143bebba64d15c6b021760a416ec022ca00..eae6482aa679680fa078626a5709a28a7da8bff0 100644
--- a/chrome/test/chromedriver/BUILD.gn
+++ b/chrome/test/chromedriver/BUILD.gn
@@ -4,7 +4,9 @@
 
 import("//build/config/python.gni")
 import("//build/config/ui.gni")
-import("//build/linux/strip_binary.gni")
+if (is_linux) {
+  import("//build/linux/strip_binary.gni")
+}
 import("//build/util/lastchange.gni")
 import("//testing/test.gni")
 import("//third_party/ffmpeg/ffmpeg_options.gni")
